src/ir_Mitsubishi.cpp

```diff
@@ -3,6 +3,9 @@
 
```

Comment:
Please add yourself here, in ir_Mitsubishi_test.cpp, and in https://github.com/markszabo/IRremoteESP8266/blob/master/.github/Contributors.md too.

You've earned it!

Reply:
I've added you to the ir_Mitsubishi* files.